isis/src/control/apps/findfeatures/FastGeom.cpp

```diff
@@ -98,8 +98,8 @@ ImageTransform *FastGeom::compute(MatchImage &query, MatchImage &train)  {
     // same size as the output image below.
     if ( tSizeFull.area()  < (m_maxarea * qSize.area()) ) {
       // std::cout << "Use direct full mapping of Train-to-Query...\n";
-      fastg.reset(new GenericTransform("FastGeomMap", t_to_q,
-                                       tSizeFull));
+      fastg.reset(new GenericTiledTransform("FastGeomMap", t_to_q,
+                                       tSizeFull, 30000));
```

Comment:
Currently hard coded to tilesize 30000.  Should this be adjusted?  Configurable?

Reply:
The limit is based on OpenCV's usage of a 16 bit int, so I think hard coded is just fine.